support-redemptiondb/cfn.yaml

```diff
@@ -0,0 +1,58 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  Stage:
+    Type: String
+    Description: Stage for the tables
+    AllowedValues:
+      - CODE
+      - PROD
+Resources:
+  RedemptionCodeDynamoTableDEV:
```

Comment:
I guess schema is all TBC at this point

Reply:
yes
 in fact we only need to specify the primary keys and other search/ordering keys in the cfn.  the rest of the data can be arbitrary.  However we will indeed probably need more PRs on the cfn.